string color
uint16 image_plane_x     # Center x value
uint16 image_plane_y     # Center y value
uint16 image_plane_r     # Radius
float32 distance_from_camera
uint8 xangle
uint8 yangle